api/src/main/java/org/apache/cloudstack/api/BaseCmd.java

```diff
@@ -185,6 +186,8 @@ public static enum CommandType {
     @Inject
     public VMSnapshotService _vmSnapshotService;
     @Inject
+    public VMScheduleManager vmScheduleManager;
```

Comment:
Avoid injecting it here and making BaseCmd even more bulkier to instatiate. Unless there is a general-purpose requirement, pl declare on VM Scheduler feature related API command classes instead.